dataproc/create_cluster_json.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+from google.cloud import dataproc_v1beta2
+import google.protobuf.json_format as json_format
+import google.cloud.dataproc_v1beta2.proto.clusters_pb2 as clusters
+
+os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = "D:\\GCP\\xxxx.json"
+
+def create_cluster(project_id,region):
+    '''
+    This function takes an external JSON with cluster configuration and creates the cluster.
+    Currently, there are no examples available to do this and I learnt it the hard way. The
```

Comment:
The second sentence of the comment is interesting to know for us, but I would remove it from the sample.